### ISSUE TITLE
Making with Sphinx HTML docs crashes

### ISSUE THREAD

**Issue by algerdnazgul** [astropy/imageutils#45]
C:\GitHub\imageutils\docs>make singlehtml
Running Sphinx v1.3.1
making output directory...

Extension error:
Could not import extension astropy.sphinx.ext.edit_on_github (exception: No modu
le named sphinx.ext.edit_on_github)

**Comment by astrofrog** [comment]
@algerdnazgul - the issue is that the Sphinx extensions should now be loaded from ``astropy-helpers``. However, I *think* this repository is no longer used since all the functionality has been moved to astropy.

**Comment by algerdnazgul** [comment]
But, in any case i think we need to correct this. How do you think?

**Comment by pllim** [comment]
Huh, I thought it is now spacetelescope/astroimutils . I have asked @larrybradley to clarify.

**Comment by cdeil** [comment]
@pllim - Yes, this is an old, inactive repo and should be removed.
@astrofrog - Can you please do it?

**Comment by larrybradley** [comment]
Yes, all of the functionality of this repo is either now in `astropy` core or in the `astroscrappy` package.  This repo should be removed.

**Comment by astrofrog** [comment]
@cdeil @larrybradley - before we remove it, can someone go through and copy the relevant issues to the astropy repository if that hasn't already been done? I can remove the repo once that is done.
